activity_main.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
Constraint layout will cause problems from phone to phone

Reply:
Should I use linear or relative layout

Reply:
Yes I Know but please merge this pull request,And you can open this issue after pull request is merged.I tried changing from Constraint Layout to Linear or Relative Layout but thats not working.So please merge this pull  request.

Reply:
How come that wont work? I haved used Linear Layout is all the apps I have built..it works like charm.
Check it and this petty thing cant be opened as an issue later on